src/utility/api.ts

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { Task } from "../types/Task.types";
+
+export const getTasks = async (): Promise<Task[]> => {
+  const response = await axios.get(`${process.env.REACT_APP_API_HOST}/tasks`);
+  return response.data.data;
```

Comment:
The `data.data` is weird/ugly, but let's focus on it later, probably it needs to be fixed in the API.

You did a good job here!

Reply:
I fixed it in the API, I just need to redeploy. After I do that I'll update accordingly here haha.

Reply:
Haha ok, well done!

Reply:
FIXED IT :D